metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -107,6 +108,15 @@ object UserConfiguration {
           |separators (even on Windows).
           |""".stripMargin
       ),
+      UserConfigurationOption(
+        "ammonite-jvm-properties",
+        """`[]`.""",
+        "['-Xmx1G']",
```

Comment:
this is invalid json, it should be `["-Xmx1G"]`

Reply:
fixed